.github/workflows/php.yml

```diff
@@ -144,6 +144,9 @@ jobs:
         -   name: Install PrestaShop
             run: php install-dev/index_cli.php --language=en --country=fr --domain=localhost --db_server=127.0.0.1 --db_password=password --db_name=prestashop --db_create=1 --name=prestashop.unit.test --email=demo@prestashop.com --password=prestashop_demo
 
+        -   name: Drop non-test database
```

Comment:
FYI: for tests, we use prefixed DB name, thus dropping the unprefixed DB name is important to make sure it is not accessed due to possibly wrong config.